src/os/bluestore/BlueStore.cc

```diff
@@ -6531,6 +6555,7 @@ void BlueStore::_txc_finish_kv(TransContext *txc)
     txc->oncommits.pop_front();
   }
```

Comment:
@liewegas 

Can I reorder oncommits callbacks with onreadable callback?
I didn't apply sync_completions to oncommits callbacks.
That's because I think that oncommits must be after onreadable.
I want to know oncommits pushed back by BlueStore::flush_commit should wait for onreadable or oncommit.

Thanks